src/app/_components/SearchBar.tsx

```diff
@@ -0,0 +1,37 @@
+import { Input, InputGroup, InputLeftElement } from '@chakra-ui/react'
+import { Dispatch, SetStateAction } from 'react'
+import { BsSearch } from 'react-icons/bs'
+
+interface Props {
+  setSearchTerm: Dispatch<SetStateAction<string>>
```

Comment:
You can use more simply:
```suggestion
  setSearchTerm: (string) => void
```